src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -78,6 +80,24 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static void fetchCitiesThroughJdbc(ClientConfig config, Properties tlsProperties) {
```

Comment:
Can we also move this code to below, so that the program follows the order we have used in the main method?

Reply:
I extracted this to a new class as per Avtar's suggestion (he wants to keep ClientWithSsl streamlined)